### DISCUSSION TITLE
SML command problem

### DISCUSSION THREAD

**Discussion by hw-schrauber** [arendst/Tasmota#18662] [Q&A]
Hi, I put together a build according to instructions for reading my smart meter. Now, I have a few things that do not run quite as expected. There is for one that when I type the command line "+1,3,s,0,9600,LK13BE,1,10,2F3F210D0A,063035310D0A" nothing happens. No more entries in the console. But I can switch to other sub-items via the main menu. If I use the line shortened "+1,3,s,0,9600,LK13BE" then I get data displayed in the UI. The part I shortened refers to the send. What exactly should the end "....,1,10,2F3F210D0A,063035310D0A" do?
There is another curiosity that may have the same cause. I called the module with the IP and I can access it, configure it, look at the data, etc.. If I then press the return arrow in the browser (FireFox), then the module crashes in such a way that only a power off/on reset allows access again. I have this effect with a Sonoff Basic as well as with a D1-mini. I have compiled the version 12.5.0 once online and once offline with VS-Code. I am a bit on the spot! Can someone help me?

**Comment by rock2ya** [comment]
Hi, i answer you, because i get some kind of same strange behavior with LK13BE
When my system boots i get no Smart Meter information shown in the Console.
If i cut the script by removing ",2F3F210D0A,063035310D0A" that section and save the script nothing happens, but if i paste it afterwars back into the script it starts reading all the desired values
